docs/src/CommutativeAlgebra/GroebnerBases/orderings.md

```diff
@@ -325,15 +327,19 @@ degrevlex([w, x])*degrevlex([y, z])
 
 ## Elimination Orderings
 
-Let $C[x]=C[x_1, \ldots, x_n]$ be a multivariate polynomial ring with coefficient ring $C$.
 Fix a subset $\sigma\subset \{1,\dots, n\}$ and write $x_\sigma$  for the set of variables $x_i$ with
 $i\in\sigma$. An *elimination ordering for $x\smallsetminus x_\sigma$*  is a monomial ordering
 $>$ on $\text{Mon}_n(x)$ which satisfies the following property: If $a$ is a monomial involving one
 of the variables in $x\smallsetminus x_\sigma$ , and $b$ is a monomial depending only on the variables in 
-$x_\sigma$, then $a > b.$ Computing a Gröbner basis of $I$ with respect to such an ordering provides one way of finding the
-intersection $I\cap C[x_\sigma]$, that is, of  *eliminating the variables in $x\smallsetminus x_\sigma$ from $I$*:
-The Gröbner basis elements which only depend on the variables in $x_\sigma$ form a Gröbner basis for
-$I\cap C[x_\sigma]$ with respect to the restriction of $>$ to the set of monomials in $I\cap C[x_\sigma]$.
+$x_\sigma$, then $a > b.$
+
+!!! note
+    The name elimination ordering stems from its application to ideals in multivariate polynomial rings:
+    Given such an ideal $I\subset C[x]=C[x_1, \ldots, x_n]$, computing a Gröbner basis of $I$ with respect
```

Comment:
```suggestion
    Given such an ideal $I\subset C[x]=C[x_1, \ldots, x_n]$, computing a Gröbner basis for $I$ with respect
```

Reply:
I usually prefer *for* as well, but use *of* in the whole chapter as do the authors of the cited Singular book
and other books on the subject (though, I likely was not completely consistent). I am happy with replacing any *of* by *for* if this can be done in an automated may. Otherwise, let us leave things as they are.